### ISSUE TITLE
Error: Expected a ReactElement, got `number`

### ISSUE THREAD

**Issue by fzaninotto** [algolia/react-element-to-jsx-string#8]
Unit test:

```js
import expect from 'expect';
import expectJSX from 'expect-jsx';
import React from 'react';

import { createRenderer } from 'react-addons-test-utils';

expect.extend(expectJSX);

describe(`toEqualJSX`, () => {
    it('should accept JSX with numbers', () => {
        const MyTable = () => (
            <span>{123}</span>
        )
        const shallowRenderer = createRenderer();
        shallowRenderer.render(<MyTable/>);
        const component = shallowRenderer.getRenderOutput();
        expect(component).toEqualJSX(<span>123</span>);
    });
});
```

Output:

```
  1) toEqualJSX should accept JSX with numbers:
     Error: react-element-to-jsx-string: Expected a ReactElement, got `number`
      at toJSXString (node_modules/expect-jsx/node_modules/react-element-to-jsx-string/index-dist.js:51:11)
      at node_modules/expect-jsx/node_modules/react-element-to-jsx-string/index-dist.js:179:12
      at Array.map (native)
      at toJSXString (node_modules/expect-jsx/node_modules/react-element-to-jsx-string/index-dist.js:104:97)
      at reactElementToJSXString (node_modules/expect-jsx/node_modules/react-element-to-jsx-string/index-dist.js:37:10)
      at Expectation.toEqualJSX (node_modules/expect-jsx/index-dist.js:27:78)
      at Context.<anonymous> (app/frontend/test/unit/components/Assessment/Test.spec.js:17:27)
```

I managed to fix it by adding another condition on type 'number' on https://github.com/algolia/react-element-to-jsx-string/blob/1e63273e554ef217f33c8ce45331dbd8a1c43ba9/index.js#L14

**Comment by vvo** [comment]
Well @fzaninotto it all look very good: you have the unit test and the fix, shoot me a PR! Or I will do the change but really you already did most of the work :)

**Comment by fzaninotto** [comment]
I was kinda hoping I wouldn't have to clone your repo...

**Comment by vvo** [comment]
Wfm. I do think the effort you put in writing the issue was longer than just submitting a PR thought :-D

**Comment by vvo** [comment]
fixed and released expect-jsx@2.1.1

```
npm install expect-jsx@2.1.1 --save-dev
```
